이은지/calculator.js

```diff
@@ -0,0 +1,19 @@
+// input에 클릭하면 해당 숫자, 문자 넣어주기
+function add(char) {
+    const display = document.getElementById('js-display');
+    display.value = display.value + char;
+}
+//! querySelector로 하면 add는 실행되는데 calculate는 실행 안 됨
+
+// eval로 입력된 값 자동 계산
+function calculate() {
+    const display = document.getElementById('js-display');
+    const result = eval(display.value);
+    document.getElementById('js-result').value = result;
+}
```

Comment:
```suggestion
//! querySelector로 하면 add는 실행되는데 calculate는 실행 안 됨

function calculate() {
  const display = document.querySelector("#js-display");
  const result = document.querySelector("#js-result");
  result.value = eval(display.value);
}
```

어떤 에러 메세지 때문에 안되셨는지 공유해 주시면 더욱 자세히 알려드릴 수 있겠지만

유추를 해보았을 때, HTML에 id로써 접근을 하도록 작성을 해주셔서 `#js-display` 처럼 `#`을 사용안하셨을 거 같아요 !

Reply:
#으로 했더니 되었어요! 감사합니다